workspaces/orchestrator/plugins/orchestrator-common/src/permissions.ts

```diff
@@ -46,7 +46,15 @@ export const orchestratorWorkflowUseSpecificPermission = (workflowId: string) =>
     },
   });
 
+export const orchestratorAdminView = createPermission({
```

Comment:
let's keep the naming conventions:
```suggestion
export const orchestratorAdminViewPermission = createPermission({
```